Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    django-user
+
+RUN mkdir -p /app/media \
+    && chown -R django-user:django-user /app \
+    && chmod -R 755 /app/media
```

Comment:
Ensure that the permissions set with `chmod` are appropriate for your application's security requirements. The current setting grants read, write, and execute permissions to the owner and read and execute permissions to others, which might be too permissive.